### PR TITLE
Include job for IPv6 testing as additional network

### PR DESCRIPTION
**Pull request by MaysaMacedo**
This commit includes a periodic job that will:

- Include additional ipv6 network to the workers
- Include additional ipv6 network to the network operator
- Enable additional traffic on the Workers
- Create Pods on different Nodes with an additional IPv6 interface
- Test IPv6 connectivity between the Pods

### CODE REVIEW

**Comment by EmilienM** [comment]
/retest
/lgtm
/hold
feel free to remove the hold when CI is green

File: ci-operator/step-registry/openstack/test/ipv6/openstack-test-ipv6-commands.sh [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        do
+            pod_name=$(echo "${i}" | awk '{print $1}')
+            pod_phase=$(echo "${i}" | awk '{print $3}')
+            if [[ "${pod_phase}" == "Running" ]]; then
+                READY=true
+            else
+                echo "Waiting for Pod ${pod_name} to be ready"
+                READY=false
+            fi
+        done <<< "$(oc -n "${IPV6_NAMESPACE}" get pods "${IPV6_POD}"--no-headers)"
```
Comment (EmilienM) [inline ci-operator/step-registry/openstack/test/ipv6/openstack-test-ipv6-commands.sh:41]: typo here, add a space

**Comment by MaysaMacedo** [comment]
Job result was green https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/32658/rehearse-32658-periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.12-e2e-openstack-additional-ipv6-network/1578387086224920576

File: ci-operator/step-registry/openstack/test/ipv6/openstack-test-ipv6-commands.sh [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+metadata:
+  name: pod-$i
+  namespace: ${IPV6_NAMESPACE}
+  annotations:
+    k8s.v1.cni.cncf.io/networks: ${ADDITIONAL_NETWORK}
+spec:
+  nodeName: ${WORKER_NAME}
+  containers:
+  - image: quay.io/kuryr/demo
+     securityContext:
```
Comment (EmilienM) [inline ci-operator/step-registry/openstack/test/ipv6/openstack-test-ipv6-commands.sh:121]: indentation issue here

**Pull request merged**